app/page.tsx

```diff
@@ -1,32 +1,29 @@
-import { Poppins } from "next/font/google";
 
+import { Poppins } from "next/font/google";
```

Comment:
Can we configure fonts on a global level , like in a generic stylesheet ? Or in the tailwind configuration ?